app/modules/users/permissions/rules.py

```diff
@@ -44,6 +44,8 @@
     ('AuditLog', AccessOperation.READ_PRIVILEGED): ['is_admin'],
     ('SocialGroup', AccessOperation.READ): ['is_researcher'],
     ('SocialGroup', AccessOperation.WRITE): ['is_researcher'],
+    ('Relationship', AccessOperation.READ): ['is_active'],
+    ('Relationship', AccessOperation.WRITE): ['is_active'],
```

Comment:
Do we really want any user to be able to create a relationship between any pair of individuals. 
In the social group code, the requestor must be a researcher and must have view permission to both individuals. 
Surely a relationship should requier the same permissions as social group?